devito/core/cpu.py

```diff
@@ -143,14 +143,15 @@ class Cpu64NoopOperator(Cpu64OperatorMixin, CoreOperator):
     def _specialize_iet(cls, graph, **kwargs):
         options = kwargs['options']
         platform = kwargs['platform']
+        compiler = kwargs['compiler']
```

Comment:
Note: not for here but long run, should be gather all of these into a `CompilerOptions` class to avoid carrying all these multiple arguments everywhere.